src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -708,7 +708,8 @@ void sofia_handle_sip_i_notify(switch_core_session_t *session, int status,
 			switch_event_add_header_string(s_event, SWITCH_STACK_BOTTOM, "subscription-expires", sip->sip_subscription_state->ss_expires);
 		}
 		if (session) {
-			switch_event_add_header_string(s_event, SWITCH_STACK_BOTTOM, "UniqueID", switch_core_session_get_uuid(session));
+			switch_channel_event_set_data(channel, s_event);
+			switch_event_add_header_string(s_event, SWITCH_STACK_BOTTOM, "Unique-ID", switch_core_session_get_uuid(session));
```

Comment:
This call is redundant.  Unique-ID is already set via switch_channel_event_set_data

Reply:
Got it. Reverting changes on adding Unique-ID